src/components/CityCreateForm/StyledCityCreate.js

```diff
@@ -16,6 +16,62 @@ export const StyledButton = styled.button`
   }
 `;
 
+export const StyledButtonWithDelete = styled.button`
+  background-color: #0d5c63;
+  border-radius: 0.6rem;
+  color: #cbf3f0;
+  font-weight: bold;
+  padding: 0.25rem 0.25rem;
+  &:hover {
+    color: #063539;
+    background-color: #2ec4b6;
+  }
+  &:active {
+    transform: scale(0.95);
+  }
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      background-color: #ccc;
+      color: #999;
+      cursor: not-allowed;
+      &:hover {
+        color: #999;
+        background-color: #ccc;
+      }
+      &:active {
+        transform: none;
+      }
+    `}
+`;
+export const StyledButtonWithDesable = styled.button`
```

Comment:
First thought is do we need this? It seems to be the same as `StyledButtonWithDelete`. (where you can pass a `disabled` prop). So it might be possible that we can delete this. If not...

typo here (see another comment with suggested code extension). Don't forget if you rename it here, to rename it wherever you import it. 

```suggestion
export const StyledButtonWithDisable = styled.button`
```